test/parallel/test-temporal-present.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+require('../common');
+
+const { ok } = require('assert');
+
+// Test verifying that Temporal is present in the global scope
+
+ok(globalThis.Temporal);
```

Comment:
Shouldn't the global leak detection in `common` be updated to include this global if it is present?

Reply:
Oh, good catch. Forgot about that.